assets/scripts/app/routing.js

```diff
@@ -1,6 +1,47 @@
-/* global URL_NEW_STREET, URL_EXAMPLE_STREET, URL_NEW_STREET_COPY_LAST, URL_SIGN_IN_REDIRECT */
-// Split from js/app/routing
-// TODO: Gradually migrate the rest of it here
+/* global URL_SIGN_IN_CALLBACK_REL, location */
```

Comment:
Similarly use `window.location` instead of having to declare it as a global variable